examples/petsc/include/petscutils.h

```diff
@@ -10,6 +10,10 @@
 #endif
 
 CeedMemType      MemTypeP2C(PetscMemType mtype);
+PetscErrorCode   VecP2C(Vec X_petsc, PetscMemType *mem_type, CeedVector x_ceed);
+PetscErrorCode   VecC2P(CeedVector x_ceed, PetscMemType mem_type, Vec X_petsc);
+PetscErrorCode   VecReadP2C(Vec X_petsc, PetscMemType *mem_type, CeedVector x_ceed);
+PetscErrorCode   VecReadC2P(CeedVector x_ceed, PetscMemType mem_type, Vec X_petsc);
```

Comment:
if you want a quick task at some point we should swap over to using these utils in the other PETSc example code (`matops.c` is probably the one?).

Reply:
Can do, I'll make a new branch